### PR TITLE
Improve help command

### LINKED ISSUE #239: Help command does not show useful information
Should be changed to show the command summary shown in the UG.

### PR DESCRIPTION
**Pull request by ZechariahTan**
Add command summary to help command window

### CODE REVIEW

**Comment by markmcwong** [comment]
LGTM

**Comment by lyueyang** [comment]
resolves #239

**Comment by lyueyang** [comment]
resolves #236

**Review by markmcwong — APPROVED** [review]
LGTM

File: src/main/resources/view/HelpWindow.fxml [hunk 1]
```diff
@@ -4,28 +4,74 @@
...
+                                       <HelpTableEntry action="delete" format="delete INDEX [INDEX INDEX...]" />
+                                       <HelpTableEntry action="pool" format="pool n/DRIVER_NAME p/DRIVER_PHONE d/TRIPDAY t/TRIPTIME c/INDEX [c/INDEX c/INDEX ...]" />
+                                       <HelpTableEntry action="unpool" format="unpool INDEX" />
+                                       <HelpTableEntry action="find" format="find PREFIX/KEYWORD [PREFIX/KEYWORD PREFIX/KEYWORD ...]" />
+                                       <HelpTableEntry action="findPool" format="findPool n/PASSENGER_NAME" />
+                                       <HelpTableEntry action="listPool" format="listPool" />
+                                       <HelpTableEntry action="list" format="list" />
+                                       <HelpTableEntry action="help" format="help" />
+                                       <HelpTableEntry action="clear" format="clear" />
+                                       <HelpTableEntry action="exit" format="exit" />
```
Comment (JoelHo) [inline src/main/resources/view/HelpWindow.fxml:54]: Perhaps having magic literals (especially in the FXML file) is not so good here

File: src/main/java/seedu/address/ui/HelpTableEntry.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
+package seedu.address.ui;
+
+/**
+ * The model for the help table shown in the help command
+ */
+public class HelpTableEntry {
```
Comment (JoelHo) [inline src/main/java/seedu/address/ui/HelpTableEntry.java:6]: I'm not sure having a wrapper class here is absolutely necessary?

**Review by JoelHo — CHANGES_REQUESTED** [review]
Do minimally remove the magic literals in the FXML

**Comment by chewterence** [comment]
merging this PR would mean that we should update the UG's help command too @ZechariahTan

**Pull request closed without merging**